doc/source/developer_guide/dg_scripts.rst

```diff
@@ -161,25 +161,25 @@ To add a new test (for example newtest), several files may be needed,
 Generating a new test, particularly the **test_newtest.script** usually takes some iteration before
 it's working properly.
 
-.. _dev_compliance:
+.. _dev_validation:
 
-Code Compliance Script
+Code Validation Script
 ----------------------
 
-The code compliance test validates non bit-for-bit model changes.  The directory 
-**configuration/scripts/tests/QC** contains scripts related to the compliance testing,
-and this process is described in :ref:`compliance`.  This section will describe a set
-of scripts that test and validate the code compliance process.  This should be done 
-when the compliance test or compliance test scripts (i.e., ``cice.t-test.py``) are modified.  
-Again, this section **documents a validation process for the compliance scripts**; it does not
-describe to how run the compliance test itself.  
+The code validation (QC) test validates non bit-for-bit model changes.  The directory 
```

Comment:
Define what QC is, if it's not defined above, e.g. "The code validation (Quality Control, QC) test..."

Reply:
Good catch, I have updated the documentation to mix validation and QC better.  I was looking at the documentation and got confused about the word compliance.  "code compliance" sounded like coding syntax, format, interfaces, and similar.  That's why I switched to "code validation".